### PR TITLE
Simplify navigation

### PR DESCRIPTION
**Pull request by javierm**
## Objectives

* Add a link in the homepage to the main poll
* Hide useless links to the poll index
* Hide useless "Menu" button on small screens

### Before these changes

The Menu button is shown on small screens

The navigation in the homepage contains just one link saying "Voting"

In the poll, there's a link to "Go back to polls"

### After these changes

The Menu button isn't shown on small screens

The navigation in the homepage is replaced with a banner showing a link to vote

There's no link to "Go back to polls" because there's only one poll

## Notes

We might need to revisit this topic after we've defined all the content that's going to be shown on the web.

### CODE REVIEW

File: app/components/custom/layout/subnavigation_component.html.erb [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<nav>
+  <ul>
+    <%= raw content_block("subnavigation_left") %>
```
[the remark this answers is not in the document]
Comment (javierm) [reply app/components/custom/layout/subnavigation_component.html.erb:3]: I know, I know :relieved:. But this is copied from the original CONSUL view, so not much we can do here.

File: db/homepage.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Widget::Card.find_or_initialize_by(header: true, cardable_id: nil, cardable_type: "SiteCustomization::Page").update!(
```
[the remark this answers is not in the document]
Comment (javierm) [reply db/homepage.rb:1]: I know, I know :relieved:. But this is copied from the original CONSUL view, so not much we can do here.

File: app/components/custom/layout/subnavigation_component.html.erb [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    <% if feature?(:help_page) %>
+      <li>
+        <%= layout_menu_link_to t("layouts.header.help"),
+                                help_path,
+                                current_page?(help_path),
+                                accesskey: "7" %>
+      </li>
+    <% end %>
+
+    <%= raw content_block("subnavigation_right") %>
```
[the remark this answers is not in the document]
Comment (javierm) [reply app/components/custom/layout/subnavigation_component.html.erb:62]: I know, I know :relieved:. But this is copied from the original CONSUL view, so not much we can do here.

**Review by taitus — APPROVED** [review]

**Review by Senen — APPROVED** [review]
👌🏼

**Pull request merged**